src/IssueViz.Security.UnitTests/Taint/TaintIssuesSynchronizerTests.cs

```diff
@@ -358,5 +373,25 @@ public TestSonarQubeIssue()
             {
             }
         }
+
+        private void SetupAnalysisInformation(Mock<ISonarQubeService> sonarQubeService, AnalysisInformation analysisInformation)
```

Comment:
It would be clearer if this parameter was called something like `mainBranchInfo`